src/server/db_slice.cc

```diff
@@ -1291,9 +1291,6 @@ pair<uint64_t, size_t> DbSlice::FreeMemWithEvictionStep(DbIndex db_ind, size_t s
 
     auto time_finish = absl::GetCurrentTimeNanos();
     events_.evicted_keys += evicted_items;
-    DVLOG(2) << "Evicted: " << evicted_bytes;
```

Comment:
why did you remove this print?

Reply:
Moved this to the [Heartbeat](https://github.com/dragonflydb/dragonfly/pull/4142/files#diff-89b55bcf480e65db9741131907708968288c37fd0993ac1680ba2eb7fb28e873R804)

Reply:
Because this callback is not always called

Reply:
oh I see you moved the prints to engine shard. ok :+1: